src/main/java/org/alfresco/rest/api/QuickShareLinks.java

```diff
@@ -133,4 +133,8 @@
     String PARAM_INCLUDE_ALLOWABLEOPERATIONS = Nodes.PARAM_INCLUDE_ALLOWABLEOPERATIONS;
 
     String PARAM_INCLUDE_PATH = Nodes.PARAM_INCLUDE_PATH;
+
+    String PARAM_INCLUDE_PROPERTIES = Nodes.PARAM_INCLUDE_PROPERTIES;
```

Comment:
As we discussed, if we can now optionally include properties then it may make sense to do the same for aspectNames (since they go hand in hand) ?

Reply:
Added